samples/cellular/tls_connect/README.md

```diff
@@ -0,0 +1,67 @@
+TLS Connect Sample Application
+==============================
+
+This example demonstrates the usage of the TLS (Transport Layer Security) API
+by giving an example of how to connect to a server that authenticates that the
+client connecting to is authorized to connect.
+
+The example demonstrates use of `ussl.wrap_socket()` with ca_certs (to only
```

Comment:
I think we're better off removing this sample altogether. `auth.pizza` isn't running anymore, and Cloudflare support told me they have no plans to put it back up. (They even took my hint and removed it from their blog post that originally described it.)

Reply:
Will do.

Reply:
Done